server/build/webpack.js

```diff
@@ -39,6 +44,10 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
       log: false,
       // required not to cache removed files
       useHashIndex: false
+    }),
+    new webpack.optimize.CommonsChunkPlugin({
```

Comment:
They might comes with the page as how it works now in next.js. 
But unlike now, it will load once.

See more info: https://webpack.js.org/plugins/commons-chunk-plugin/